src/FffGcodeWriter.cpp

```diff
@@ -1468,63 +1466,65 @@ bool FffGcodeWriter::processSingleLayerInfill(const SliceDataStorage& storage, L
     {
         return false;
     }
-    const coord_t infill_line_distance = mesh.settings.get<coord_t>("infill_line_distance");
```

Comment:
Clang tidy warns against this, since the get specifies the type name in the template all ready.
Since auto isn't dynamically determined but known at compile time. I'm tempted to leave it as auto otherwise the code base will never be modernized.

But if you feel strongly about it I have no problems returning it to hard coded type.